### PR TITLE
[feat/fix]: Minor improvements in the YouTube description file

### PR DESCRIPTION
**Pull request by Panquesito7**
#### What does this PR do?

- Improves the YouTube `review.md` file.
  - Removed Markdown stuff such as headers. Other minor improvements are shown in the Git diff.

#### Description of the task to be completed?

None.

#### How can this be manually tested?

I don't think this applies here.

#### Any background context you want to provide?

Nope.

#### Is there any relevant issue to this PR?

#2202

#### Questions

None.

### CODE REVIEW

File: youtube/reviews.md [hunk 1]
```diff
@@ -30,18 +30,18 @@ Nicholas Carrigan https://github.com/nhcarrigan
...
+Follow on other socials for behind the scenes footage. Join the Discord server to continue the conversation.
 
 http://eddiejaoude.io 
 https://github.com/eddiejaoude
 http://youtube.com/eddiejaoude
 http://twitter.com/eddiejaoude
 http://instagram.com/eddiejaoude
 http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
+https://discord.eddiehub.org/
```
Comment (schmelto) [inline youtube/reviews.md:46]:
```suggestion
http://discord.eddiehub.org/
```
Https isn't working on this link

File: youtube/reviews.md [hunk 2]
```diff
@@ -30,18 +30,18 @@ Nicholas Carrigan https://github.com/nhcarrigan
...
 
-## FIND ME ON OTHER SOCIAL PLATFORMS
+FIND ME ON OTHER SOCIAL PLATFORMS
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
+Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!
 https://www.youtube.com/eddiejaoude?sub_confirmation=1
 
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+Follow on other socials for behind the scenes footage. Join the Discord server to continue the conversation.
```
Comment (schmelto) [inline youtube/reviews.md:38]:
```suggestion
Follow on other socials for behind the scenes footage and join the Discord server to continue the conversation.
```
For me this sounds better but you don't have to commit this.

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
Lgtm

@eddiejaoude maybe you should also take a look here

**Review by krishdevdb — APPROVED** [review]

**Comment by krishdevdb** [comment]
Well, merging this pr makes an issue open, we need to be consistent so we shall also change the other files? This is a good first issue for someone. (Open an issue, then you might also open a pr)...

**Review by KukretiShubham — APPROVED** [review]

**Comment by KukretiShubham** [comment]
LGTM👍

**Review by eddiejaoude — APPROVED** [review]
LGTM

**Comment by eddiejaoude** [comment]
Thank you all 👍 great collaboration 💥

**Pull request merged**